python/python/glide/async_commands/core.py

```diff
@@ -1202,6 +1224,42 @@ async def zcard(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.Zcard, [key]))
 
+    async def zcount(
+        self,
+        key: str,
+        min_score: Union[InfBound, BoundPair],
+        max_score: Union[InfBound, BoundPair],
+    ) -> int:
+        """
+        Returns the number of members in the sorted set stored at `key` with scores between `min_score` and `max_score`.
+
+        See https://redis.io/commands/zcount/ for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            min_score (Union[InfBound, BoundPair]): The minimum score to count from.
+                Can be an instance of InfBound representing positive/negative infinity,
+                or BoundPair representing a specific score and inclusivity.
+            max_score (Union[InfBound, BoundPair]): The maximum score to count up to.
+                Can be an instance of InfBound representing positive/negative infinity,
+                or BoundPair representing a specific score and inclusivity.
+
+        Returns:
+            int: The number of members in the specified score range.
+
+        Examples:
+            >>> await zcount("my_sorted_set", BoundPair(5.0 , true) , InfBound.POS_INF)
+                2  # Indicates that there are 2 members with scores between 5.0 (not exclusive) and +inf in the sorted set "my_sorted_set".
+            >>> await zcount("my_sorted_set", BoundPair(5.0 , true) , BoundPair(1.0 , false))
+                1  # Indicates that there is one member with 5.0 < score <= 10.0 in the sorted set "my_sorted_set".
```

Comment:
in the example you wrote 1., in the comment  10.0. 
I guess that it's a typo, but what happens if max_score < min_score?

Reply:
what happens if key isn't exists? an error returned, or a null?

Reply:
If `key` does not exist, it is treated as an empty sorted set, and the command returns 0.
If `max_score` < `min_score`, 0 is returned.
Added this to the doc.